decidim-assemblies/app/models/decidim/assembly.rb

```diff
@@ -48,5 +57,13 @@ def hashtag
     def to_param
       slug
     end
+
+    def self.private_assemblies
+      where(private_space: true)
+    end
+
+    def self.non_private_assemblies
```

Comment:
This class already extends `Participable`, which includes some functionality to figure out `public` spaces, as a means to provide statistics on the home page and other places. We probably should be using that instead (probably overriding it for the `assemblies` and `processes` case):

https://github.com/decidim/decidim/blob/master/decidim-core/lib/decidim/participable.rb#L82

Otherwise, data from private spaces might leak to statistics.

Reply:
If I didn't go wrong. It was said that statistics would not take into account private processes, no?